cmd/cli/commands/infrastructure.go

```diff
@@ -35,190 +28,190 @@ var createInfraCmd = &cobra.Command{
-		jsonFile, _ := cmd.Flags().GetString("file")
-		if jsonFile == "" {
-			fmt.Println("Error: JSON file not provided")
-			os.Exit(1)
-		}
-		if err := validateFilePath(jsonFile); err != nil {
-			fmt.Printf("Error validating file path: %v\n", err)
-			os.Exit(1)
-		}
-		// #nosec G304 -- file path is validated before use
-		data, err := os.ReadFile(jsonFile)
-		if err != nil {
-			fmt.Printf("Error reading JSON file: %v\n", err)
-			os.Exit(1)
-		}
-
-		if err := json.Unmarshal(data, &req); err != nil {
-			fmt.Printf("Error parsing JSON file: %v\n", err)
-			os.Exit(1)
-		}
-
-		// Validate that instances array is not empty
-		if len(req.Instances) == 0 {
-			fmt.Println("Error: No instances specified in the JSON file")
-			os.Exit(1)
-		}
-
-		jsonData, err := json.Marshal(req)
-		if err != nil {
-			fmt.Printf("Error creating request: %v\n", err)
-			os.Exit(1)
-		}
-
-		// Create a new request
-		httpReq, err := http.NewRequest("DELETE", "http://localhost:8080/api/v1/instances",
-			bytes.NewBuffer(jsonData))
-		if err != nil {
-			fmt.Printf("Error creating request: %v\n", err)
-			os.Exit(1)
-		}
-		httpReq.Header.Set("Content-Type", "application/json")
-
-		// Send the request
-		client := &http.Client{}
-		resp, err := client.Do(httpReq)
-		if err != nil {
-			fmt.Printf("Error deleting infrastructure: %v\n", err)
-			os.Exit(1)
-		}
-		defer func() {
-			if cerr := resp.Body.Close(); cerr != nil {
-				err = fmt.Errorf("error closing response body: %w", cerr)
-			}
-		}()
-
-		var result interface{}
-		if err := json.NewDecoder(resp.Body).Decode(&result); err != nil {
-			fmt.Printf("Error decoding response: %v\n", err)
-			os.Exit(1)
-		}
-
-		prettyJSON, _ := json.MarshalIndent(result, "", "  ")
-		fmt.Println(string(prettyJSON))
+		fmt.Println("BROKEN")
+		os.Exit(1)
+		// var req infrastructure.DeleteInstanceRequest
+
+		// // Check if JSON file is provided
+		// jsonFile, _ := cmd.Flags().GetString("file")
+		// if jsonFile == "" {
+		// 	fmt.Println("Error: JSON file not provided")
+		// 	os.Exit(1)
+		// }
+		// if err := validateFilePath(jsonFile); err != nil {
+		// 	fmt.Printf("Error validating file path: %v\n", err)
+		// 	os.Exit(1)
+		// }
+		// // #nosec G304 -- file path is validated before use
+		// data, err := os.ReadFile(jsonFile)
+		// if err != nil {
+		// 	fmt.Printf("Error reading JSON file: %v\n", err)
+		// 	os.Exit(1)
+		// }
+
+		// if err := json.Unmarshal(data, &req); err != nil {
+		// 	fmt.Printf("Error parsing JSON file: %v\n", err)
+		// 	os.Exit(1)
+		// }
+
+		// // Validate that instances array is not empty
+		// if len(req.Instances) == 0 {
+		// 	fmt.Println("Error: No instances specified in the JSON file")
+		// 	os.Exit(1)
+		// }
+
+		// jsonData, err := json.Marshal(req)
+		// if err != nil {
+		// 	fmt.Printf("Error creating request: %v\n", err)
+		// 	os.Exit(1)
+		// }
+
+		// // Create a new request
+		// httpReq, err := http.NewRequest("DELETE", "http://localhost:8080/api/v1/instances",
```

Comment:
it is out of the scope of this PR but for the CLI commands, I would call the services directly instead of requiring to have the API server to be running and calling its endpoints.

Reply:
yes I have an issue already for fixing the CLI